exp-player/addon/components/exp-rating-form/component.js

```diff
@@ -552,7 +551,14 @@ const Validations = buildValidations(generateValidators(questions));
 export default ExpFrameBaseComponent.extend(Validations, {
     type: 'exp-rating-form',
     layout: layout,
-    questions: questions,
+    questions: Ember.computed(function() {
+        var profileId = this.get('session').get('profileId').split('.')[1];
+        var participantId = profileId.split('_')[1];
+        if (participantId % 2 === 0) {
```

Comment:
Preferable to grab this value from the session rather than recomputing. If we every change how conditions are assigned it would be nice to know only one place in the code needs updating.